kubetest/kind.go

```diff
@@ -0,0 +1,177 @@
+/*
```

Comment:
we should move this into a subpackage of kubetest for better isolation. the `dind` impl did this as well